bin/fontbakery-check-font-version.py

```diff
@@ -0,0 +1,85 @@
+"""
+fontbakery-check-font-version.py:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Check the version number of a family hosted on fonts.google.com.
+
+`fontbakery check-font-version.py "Roboto"`
+
+
+Comparison against a local family can be made using the -lc argument:
+
+`fontbakery check-font-version.py "Roboto" -lc [/dir/Roboto-Regular.ttf, ...]`
```

Comment:
The fontbakery subcommands code will accept the following:

> fontbakery check-font-version "Roboto" -lc [/dir/Roboto-Regular.ttf, ...]

Please remove the **.py** suffix here.